spec/avromatic/model/builder_spec.rb

```diff
@@ -141,6 +141,21 @@
       end
 
       it_behaves_like "a generated model"
+
+      context "with a specified schema subject" do
+        let(:schema_subject) { 'test.primitive_types-value' }
```

Comment:
nit: Some of this test data might be easier to understand if we use the `-subject` suffix rather than `-value` (especially for the key schemas).